packages/ibm-products/src/components/EditInPlace/EditInPlace.tsx

```diff
@@ -443,6 +449,11 @@ EditInPlace.propTypes = {
    * handler that is called when the save button is pressed or when the user removes focus from the input if it has a new value
    */
   onSave: PropTypes.func.isRequired,
+  /**
+   * Placeholder for text input
+   */
+  /**@ts-ignore*/
```

Comment:
```suggestion
```
as we make the prop optional in the interface, this can be removed, which shouldn't throw any error now.